src/SDKs/HealthcareApis/Management.HealthcareApis/Microsoft.Azure.Management.HealthcareApis.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HealthcareApis</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure Healthcare Apis.</Description>
+    <AssemblyTitle>Microsoft Azure Healthcare Apis Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.HealthcareApis</AssemblyName>
+    <Version>1.0.5-preview</Version>
+    <RootNamespace>Microsoft.Azure.Management.HealthcareApis</RootNamespace>
+    <PackageTags>Microsoft Azure Healthcare Apis;HealthcareApis management;HealthcareApis;</PackageTags>
+		<PackageReleaseNotes>Management SDK for Healthcare APIs.</PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+		<TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+		<SignAssembly>false</SignAssembly>
+	</PropertyGroup>
+	
```

Comment:
miss PackageReleaseNotes?

Reply:
Line 14 specifies the PackageReleaseNotes. I will fix the intendation though.

Reply:
Please format this file.